XLS-0070d-credentials/README.md

```diff
@@ -159,13 +257,13 @@ The list has a minimum size of 1 and a maximum size of 8 credentials.
 |`Issuer`|✔️|`string`|`AccountID`|The issuer of the credential.|
 |`CredentialType`|✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
 
```

Comment:
Mention about there will be no duplicates in array?

Reply:
Does it error if there are duplicates, or just silently remove them?